modules/thunder_taxonomy/thunder_taxonomy.install

```diff
@@ -87,3 +70,29 @@ function thunder_taxonomy_update_8002() {
   // Output logged messages to related channel of update execution.
   return $thunderUpdater->logger()->output();
 }
+
+/**
+ * Use core's term status field definition.
+ */
+function thunder_taxonomy_update_8003() {
```

Comment:
We have a problem with taxonomy_update_8601(). Our users will get a message which if they followed the instructions would result in additional unnecessary fields on their taxonomy. See https://www.drupal.org/project/drupal/issues/2991346